docs_src/async_sql_databases/tutorial001.py

```diff
@@ -58,8 +58,35 @@ async def read_notes():
     return await database.fetch_all(query)
 
 
+@app.get("/notes/{id}/", response_model=Note)
+async def read_one_note(id: int):
+    query = notes.select().where(notes.c.id == id)
```

Comment:
why do `notes.c.id` instead of `notes.id`, some specific databases thing?

Reply:
yes, `c` stands for column here. this is a sqlalchemy interface.

Reply:
Yes, I also have updated the docs to have a short explanation about it.